### DISCUSSION TITLE
Whats the difference between @changed and @data-model-changed ?

### DISCUSSION THREAD

**Discussion by HFolkertH** [cincheo/daquota#38] [Q&A]
Hello Renaud

I have the feeling that they react differently when I change a component value. 
A Select field has a @change and a @data-model-changed event, but a datepicker only has a  @data-model-changed event. Why?

kind regards
Folkert

**Comment by renaudpawlak** [comment]
The difference is subtle...
``@change`` means that the user changes the value of the select by explicitly doing it with the UI, it will also trigger a ``@change-data-model`` event. ``@change-data-model`` is triggered when the data model change in any circumstances, including when using a ``setData`` action or trough user interaction.

The datapicker does not support ``@change`` because the component does not support it natively. I guess we can live without it :)

**Reply by HFolkertH** [reply]
Hello Renaud

It looks like @Change-data-model fires on every change in the datamodel and not only to a change in the field that i have bound it to it through a field-event. 

I need to act on a UI change in a datepicker-field and since there is no @chance event on a datepicker-field, i have a small problem.  Do you know  a workaround?

kind regards
Folkert
